monitoring/osd-understanding-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,29 @@
+[id="understanding-the-monitoring-stack"]
+= Understanding the monitoring stack
+include::modules/attributes-openshift-dedicated.adoc[]
+:context: understanding-the-monitoring-stack
+
+toc::[]
+
+In {product-title}, you can monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. You can monitor your own projects without the need for an additional monitoring solution.
+
+[NOTE]
+Follow the instructions in this chapter carefully to configure a supported Prometheus instance. Custom Prometheus instances are not supported by {product-title}.
+
+// Understanding the monitoring stack
+include::modules/osd-monitoring-understanding-the-monitoring-stack.adoc[leveloffset=+1]
+include::modules/osd-monitoring-components-for-monitoring-user-defined-projects.adoc[leveloffset=+2]
+include::modules/osd-monitoring-targets-for-user-defined-projects.adoc[leveloffset=+2]
+
+[id="understanding-the-monitoring-stack-additional-resources"]
+== Additional resources
+
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc#accessing-monitoring-for-user-defined-projects[Accessing monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/4.7/monitoring/understanding-the-monitoring-stack.html#default-monitoring-components_understanding-the-monitoring-stack[Default monitoring components]
+* link:https://docs.openshift.com/container-platform/4.7/monitoring/understanding-the-monitoring-stack.html#default-monitoring-targets_understanding-the-monitoring-stack[Default monitoring targets]
+* link:https://docs.openshift.com/container-platform/4.7/support/remote_health_monitoring/about-remote-health-monitoring.html[About remote health monitoring]
```

Comment:
This links to OCP 4.7 doc, is it needed?

Reply:
It is informational only. The links are a replacement for the information that I removed from this doc because it seemed confusing to keep too much default monitoring information since we're focusing on workloads for user-defined projects.